src/app/modules/main/communities/models/curated_community_item.nim

```diff
@@ -102,4 +110,10 @@ proc setPermissionModelItems*(self: CuratedCommunityItem, items: seq[TokenPermis
   self.permissionModel.setItems(items)
 
 proc getAmIBanned*(self: CuratedCommunityItem): bool =
-  return self.amIBanned
\ No newline at end of file
+  return self.amIBanned
+
+proc getJoined*(self: CuratedCommunityItem): bool =
+  return self.joined
+
+proc getEncrypted*(self: CuratedCommunityItem): bool =
+  return self.encrypted
```

Comment:
You can omit the `return` keyword

Reply:
it's more or less inline with the style used in each file. Here is `return self.x`, in `src/app_service/service/message/dto/status_community_link_preview.nim` is something like `result = self.x`